amulet/level/formats/anvil_forge_world.py

```diff
@@ -33,5 +39,15 @@ def game_version_string(self) -> str:
         except Exception:
             return f"Java Forge Unknown Version"
 
+    def _register_modded_dimensions(self):
+        for path, directories, files in os.walk(os.path.join(self.path, "dimensions")):
+            if set(("data", "entities", "poi", "region")).issubset(set(directories)):
+                dimension_path_split = path.split(os.sep)
+                dimensions_directory_index = dimension_path_split.index("dimensions")
+                dimension_name = f"{dimension_path_split[dimensions_directory_index + 1]}:{'/'.join(dimension_path_split[dimensions_directory_index + 2:])}"
```

Comment:
You should probably add a check if we are at least two directories deep otherwise this will error